Resources/Textures/DeltaV/Clothing/Head/Hats/blondegigapompodour.rsi/meta.json

```
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from Monkestation2.0 https://github.com/Monkestation/Monkestation2.0/blob/master/monkestation/icons/mob/sprite_accessories/hair.dmi. Edited by @ps3moira#9488",
```

Use permalinks (links to the file on a certain commit) so people never get a 404 if the file gets moved.
```suggestion
  "copyright": "Taken from Monkestation2.0 https://github.com/Monkestation/Monkestation2.0/blob/8f95f22bc07f50f381b683fe54763268af034825/monkestation/icons/mob/sprite_accessories/hair.dmi and edited by @ps3moira#9488",
```